docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Ingress Controller
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc kubernetes Ingress reverse-proxy
+---
+
+# Kubernetes Ingress Controller
+
+Use Pomerium as a first-class secure-by-default Ingress Controller. Dynamically provision routes from Ingress resources and set policy based on annotations. By defining routes as Ingress resources you can independently create and remove them from Pomerium's configuration. This enables workflows more native to Kubernetes environments such as actions based on pull requests.
+
+## Prerequisites
+
+- A certificate management solution. If you do not already have one in place, this article covers using Cert Manager.
+- A Redis backend with high-persistence is highly recommended.
+
+### System Requirements
+
+- Kubernetes v1.19.0+
+- Pomerium [Helm Chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) v25.0.0+
+
+### Limitations
+
+::: warning
+
+Only one Ingress Controller instance/replica is supported per Pomerium cluster.
```

Comment:
Is an instance different from a replica?

Reply:
I don't know the answer. @travisgroth?

Reply:
Yes but we also limit to one active replica due to election semantics and the way rollouts work.

Reply:
@travisgroth so is this the most clear way of saying that? Should it be "instance and/or replica"?

Reply:
when we create an Ingress Controller deployment via the helm chart, we specify it only has one replica: 

```yaml
apiVersion: apps/v1
kind: Deployment
spec:
  replicas: 1
```

Reply:
So... Should we just say "replica" and remove "instance"? Or how about:

```suggestion
Only one Ingress Controller instance (defined in the spec as a replica) is supported per Pomerium cluster.
```
?

Reply:
Probably best to keep it simple:

```suggestion
Only one Ingress Controller instance is supported per Pomerium cluster.
```